### ISSUE TITLE
[Bug] MenuItem @click handlers called twice

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#1615]
### Problem

Almost anywhere there is a `menu-item` with a `@click` handler, e.g.:

```pug
                menu-item(
                  v-if='item.status.isActive'
                  tag='button'
                  item-id='revoke'
                  icon='times'
                  @click='handleRevokeClick(item.id)'
                )
                  i18n Revoke Link
```

The handler gets called twice when the menu item is clicked.

I tested this with `Revoke Link` above and also `Payment Details`:

```pug
      menu-item(
        tag='button'
        item-id='message'
        icon='info'
        @click='openModal("PaymentDetail", { id: payment.hash, period: payment.period })'
      )
        i18n Payment details
```

The `openModal` call is called twice.

Interestingly, the `action` in `MessageActions.vue` does not seem to be called twice:

```pug
      menu-item.hide-desktop.is-icon-small(
        tag='button'
        v-if='isEditable'
        @click='action("editMessage")'
      )
        i.icon-pencil-alt
        i18n Edit
```

I have no idea why sometimes they're called twice and other times they aren't.

### Solution

Debug the `MenuItem.vue` component to figure out why this is happening and fix.

```vue
<template lang='pug'>
//
  MenuItem is a wrapper around ListItem with extra styles
  and to update MenuSelect when an option was selected.
  QUESTION: Is v-bind='$attrs' the correct way to pass down
  the props and slot from MenuItem to ListItem?
list-item.c-menuItem(disable-radius='' v-bind='$attrs' v-on='$listeners' @click='handleSelect')
  slot
</template>
<script>
import ListItem from '@components/ListItem.vue'
export default ({
  name: 'MenuItem',
  inject: ['Menu'],
  components: { ListItem },
  inheritAttrs: false,
  methods: {
    handleSelect () {
      this.$listeners.click && this.$listeners.click()
      this.Menu.handleSelect(this.$attrs['item-id']) // Use $attrs to access passed props
    }
  }
}: Object)
</script>
```

cc @pieer @SebinSong @Silver-IT @sandrina-p (if any of you have an idea of why this my be happening please comment below)

**Comment by SebinSong** [comment]
@taoeffect I just tried them myself and wasn't able to encounter the bug.
It seems like the latest merge of [this change](https://github.com/okTurtles/group-income/commit/f80da54a31cc62e195bf3658fc4f10063207137d#diff-1875253ac367814652e1a7cff5c8067fb18e2bb5a72f2a325d48301a3ff946c8) by Alex has fixed it.

**Comment by taoeffect** [comment]
🤯 

Wow you're right! Great job @Silver-IT! 😄 👍 

Closed by @Silver-IT in PR  #1589
